composeApp/src/commonMain/kotlin/domain/usecase/GetEventInfo.kt

```diff
@@ -0,0 +1,4 @@
+package domain.usecase
+
+class GetEventInfo {
```

Comment:
Here we can move forward with the implementation, what do you think?

Something like:
```kotlin
class GetEventInfo(
   private val eventsRepository: EventsRepository
) {
     fun execute(eventId: String): Event {
          return eventsRepository.getEventInfo(eventId)
     }
}
```

Reply:
done